### ISSUE TITLE
Windows Server 2012 Crash when Java FX starts

### ISSUE THREAD

**Issue by tresf** [qzind/tray#924]
Note, this issue is not reproducible on a fresh install, but was present on a Xeon CPU running on an AHV Nutanix virtualized environment with Adoptium and AdoptOpenJDK with QZ Tray 2.1.5.

```
Faulting application name: javaw.exe, version: 11.0.11.0, time stamp: 0x607f471a
Faulting module name: ucrtbase.DLL, version: 10.0.17134.12, time stamp: 0x587decd7
Exception code: 0xc0000409
Fault offset: 0x000000000006e75e
Faulting process id: 0x2b2c0
Faulting application start time: 0x01d82d92b3f17d62
Faulting application path: C:\Program Files\AdoptOpenJDK\jdk-11.0.11.9-hotspot\bin\javaw.exe
Faulting module path: C:\Program Files\AdoptOpenJDK\jdk-11.0.11.9-hotspot\bin\ucrtbase.DLL
Report Id: 8296f45a-9986-11ec-838a-e2e70c4793f1
Faulting package full name: 
Faulting package-relative application ID: 
```

A temporary stop-gap for this hard-crash is to provide a system property/override to not initialize JavaFX on startup.

@bberenz can you add the property to 2.1 (we can later cherry-pick to 2.2).

**Comment by tresf** [comment]
We've tested QZ Tray 2.1.5 with the Java FX task on the following:

✅  Windows Server 2012 R2 Build 9600 in VirtualBox
✅  Windows Server 2012 R2 Build 9600 on Amazon EC2
✅  Windows Server 2012 R2 Build 9600 on bare metal
✅  Windows Server 2012 R2 Build 9600 on Nutanix

All environments used Xeon processors, none crashed.

I'm closing this as invalid until we have steps to reproduce.

**Comment by tresf** [comment]
Reopening... 

I was given an export of all printer mappings on the crashing server, imported it into Windows Server 2012 and the issue is 100% reproducible.

It seems the issue occurs when the printer searching takes a very long time (e.g. 5 minutes) and JavaFX then crashes during its capture.

**Comment by tresf** [comment]
This issue is quite odd.  As it turns out, something with fetching `PrintResolution` for 1,400 printers (specific ones imported from the client environment) followed by starting JavaFX crashes the runtime.

✅ If we start JavaFX first, no crash.
✅ If we fetch these `PrintResolution` for a percentage (e.g. 300), no crash.
✅ If we don't fetch the `PrintResolution` at all, no crash.

So this has resulted in newly created #932, which ultimately will workaround this crash by only populating the `PrintResolution` when specifically requested (lazy fetching).  This has the advantage of speeding up printer searching by a factor of 1,600% and fixing the crash in the target environment.

These improvements are staged for 2.1.6 / 2.2.2.

**Comment by tresf** [comment]
> If we start JavaFX first, no crash.

This is the fix we decided to go with.  Fixed via #934, #936.
